src/components/developer-preview.js

```diff
@@ -5,7 +5,7 @@ class DeveloperPreview extends React.Component {
     return (
       <div>
         <p className='ds-text'>The Developer Preview allows developers to build direct integrations with other software to test programmatically submitting Quality Payment Program (QPP) performance data to CMS. The API’s functionality in the Developer Preview closely matches that of the QPP system that is in production.</p>
-        <p className='ds-text'>The Submissions API base URL for the Developer Preview is: <strong><a href='https://preview.qpp.cms.gov/api/submissions'>preview.qpp.cms.gov/api/submissions</a></strong></p>
+        <p className='ds-text'>The Submissions API base URL for the Developer Preview is: <strong>https://preview.qpp.cms.gov/api/submissions/</strong></p>
```

Comment:
Wouldn't we still want this as a link?

Reply:
I think we removed the link because its the base link is not the preview site. It tripped me up and I thought there was something wrong with kong.